nixos/modules/services/web-apps/coder.nix

```diff
@@ -0,0 +1,155 @@
+{ config, lib, options, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.coder;
+  name = "coder";
+in {
+  options = {
+    services.coder = {
+      enable = mkEnableOption (lib.mdDoc "Coder service");
+
+      user = mkOption {
+        type = types.str;
+        default = "coder";
+        description = lib.mdDoc ''
+          User under which the coder service runs.
+        '';
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "coder";
+        description = lib.mdDoc ''
+          Group under which the coder service runs.
```

Comment:
This should contain that it is automatically added if left the default.

Reply:
Like [this](https://github.com/NixOS/nixpkgs/blob/5f588eb4a958f1a526ed8da02d6ea1bea0047b9f/nixos/modules/services/mail/maddy.nix#L158).